reflex-algos/pom.xml

```diff
@@ -18,6 +18,7 @@
         <shaded.prefix>com.parallelmachines.shaded</shaded.prefix>
         <scala.binary.version>2.11</scala.binary.version>
         <reflexVersion>1.5.0</reflexVersion>
+        <skipMlopsTests>false</skipMlopsTests>
```

Comment:
Maybe `skipPyTests`, because it skips both *mlops* and *mlcomp*

Reply:
did it differently...
exec=maven-plugin supports -Dexec.skip to skip whole plugin execution.
Expand this to be able to provide -Dexec.skip=test